lesson18/src/main/java/com/lesson18/entity/EmployeeDao.java

```diff
@@ -0,0 +1,17 @@
+package com.lesson18.entity;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface EmployeeDao {
+
+    void addEmployee(Employee employee);
+
+    void editEmployee(Employee employee, int employeeId);
```

Comment:
Could we put emp;oyeeId into object employee?
![image](https://user-images.githubusercontent.com/20267148/57340524-c6e02480-713e-11e9-8143-8f33392a3ad5.png)